packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -38,12 +83,12 @@ $SearchBox-height: 36px;
 .rootIsActive {
   border-color: $ms-color-themeDarker;
 
-  .field {
-    @include padding-left(8px);
-  }
-
   .icon {
-    display: none;
+    flex-basis: 4px;
+    transition: flex-basis $ms-duration1;
+     & > * {
+      display: none;
```

Comment:
back to this...i did it for a couple reasons. I can't simply display none the icon container as that'd break the animation. But I need to hide the contents of the icon container. Right now that's an expected class name, but if we ever provide a method for overriding the icon, that class might change. 

The only other option would be to wrap the rendered icon in yet another div, and target that. This seemed simpler.